.rubocop.yml

```diff
@@ -0,0 +1,46 @@
+AllCops:
+  DisplayCopNames: true
+  Exclude:
+   - 'bin/*'
+   - 'db/schema.rb'
+   - 'vendor/bundle/**/*'
+   - 'spec/dummy/**/*'
+  Include:
+    - 'lib/tasks/**/*'
+    - '**/Gemfile'
+    - '**/Rakefile'
+    - '**/Guardfile'
+  TargetRubyVersion: 2.3
```

Comment:
@Argonus of course. Good catch!

I just noticed that gemspec specifies ruby >= 2.2.2. Which one should be the lowest supported ruby vesion?

https://github.com/spark-solutions/spree_shopify_importer/blob/master/spree_shopify_importer.gemspec#L13